sdv/metadata/single_table.py

```diff
@@ -35,9 +35,8 @@ class SingleTableMetadata:
     }
 
     _NUMERICAL_REPRESENTATIONS = frozenset([
-        'int', 'int64', 'int32', 'int16', 'int8',
-        'uint', 'uint64', 'uint32', 'uint16', 'uint8',
-        'float', 'float64', 'float32', 'float16', 'float8',
+        'Float', 'Int64', 'Int32', 'Int16', 'Int8',
```

Comment:
Changing these to match RDT. Not sure about the naming scheme though, whether to use upper case letters or not, or maybe we should allow both (also, I'm removing 'int' and 'uint' since they don't mean anything, but maybe we could keep them as aliases for 'Int64'/'UInt64')? I'll leave as is for now, it shouldn't matter much either way, I imagine this parameter is not used frequently.